src/main/java/com/es/phoneshop/web/tools/RequestTools.java

```diff
@@ -0,0 +1,13 @@
+package com.es.phoneshop.web.tools;
+
+import javax.servlet.http.HttpServletRequest;
+import java.text.DecimalFormat;
+import java.text.NumberFormat;
+import java.text.ParseException;
+
+public class RequestTools {
+    public static Integer parseIntegerUsingLocale(HttpServletRequest request, String number) throws ParseException {
+        NumberFormat decimalFormat = DecimalFormat.getInstance(request.getLocale());
```

Comment:
That's great that you are using locale.
Can we also add some check for non-negative numbers, if necessary?